### ISSUE TITLE
PyCharm SciView: NaN values are handled incorrectly

### ISSUE THREAD

**Issue by alshan** [JetBrains/lets-plot#282]
When numeric series contains some NaN values, the entire series is treated as 'categorical':

```
(ggplot() + geom_point(aes(x=[d for d in range(100)]+[float('NaN')]))).show()
```
produces plot:
![image](https://user-images.githubusercontent.com/4430922/103799041-8a05b880-5018-11eb-9bcc-cb3e4c47ae7a.png)

while, of course, the expected plot is:
![image](https://user-images.githubusercontent.com/4430922/103799190-c20cfb80-5018-11eb-83d3-c640f294ae67.png)

**Comment by alshan** [comment]
Fixed by this commit: https://github.com/JetBrains/lets-plot/commit/616e0b42175697cf1b0a4444c28a45e2f2046f09

Will take effect after the next release of Lets-plot.

**Comment by alshan** [comment]
Fixed in [v2.0.0](https://github.com/JetBrains/lets-plot/releases/tag/v2.0.0)
